packages/components/src/Notification/Notification.module.scss

```diff
@@ -34,18 +34,21 @@ $tc-notification-icon-size: $svg-md-size !default;
 	}
 
 	&-info,
+	&-info p,
```

Comment:
Could we rely on className [`tc-notification-message`](https://github.com/Talend/ui/blob/master/packages/components/src/Notification/Notification.component.js#L67) instead of element type?

Reply:
👍

Reply:
Done :white_check_mark: